src/react-decor/index.ts

```diff
@@ -2,20 +2,19 @@ export {decorReactClass} from "./react-decor-class";
 export {DecorReactHooks, StatelessDecorReactHooks, StatefulElementHook, StatelessElementHook} from "./common";
 
 import {Component, ComponentType} from "react";
-import {createDecorationReflection} from "./react-decor-reflection";
-import {privateState} from "../core/private-state";
-import {StatefulElementHook, StatelessElementHook} from "./common";
+import {reflection} from "./react-decor-reflection";
 import {decorReactClass} from "./react-decor-class";
 import {DecorReactHooks, isReactClassComponent, StatelessDecorReactHooks} from "./common";
 import {decorReactFunc} from "./react-decor-function";
 
-export const decorationReflection = createDecorationReflection('react-decor-private-state');
+const decorationReflection = reflection('react-decor-private-state');
```

Comment:
think how you want this state to be indexed when reviewing a class in dev mode
`'react-decor-private-state'` seems lacking